FBC/FinancialInstruments/InstrumentPricing.rdf

```diff
@@ -0,0 +1,478 @@
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;BestOffer">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-ip;OfferPrice"/>
+		<rdfs:label xml:lang="en">best offer</rdfs:label>
+		<skos:definition xml:lang="en">lowest price acceptable to a prospective seller for a given security at a particular point in time</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;BidAskSpread">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-ip;PriceSpread"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-utl-alx;hasArgument"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-fi-ip;BidPrice"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-utl-alx;hasArgument"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-fi-ip;OfferPrice"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">bid ask spread</rdfs:label>
+		<skos:definition xml:lang="en">difference between an offer (ask) price and a bid price</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;BidPrice">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-ip;SecurityPrice"/>
+		<rdfs:label xml:lang="en">bid price</rdfs:label>
+		<owl:disjointWith rdf:resource="&fibo-fbc-fi-ip;OfferPrice"/>
+		<skos:definition xml:lang="en">price a prospective buyer is willing to pay</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">The term &apos;bid price&apos; is used by traders / market makers with respect to a given security, and that are prepared to buy or sell round lots at publicly quoted prices, and by specialists in certain instruments that perform similar functions on an exchange.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;ClosingPriceDeterminationMethod">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-gao-obj;Strategy"/>
+		<rdfs:label xml:lang="en">closing price determination method</rdfs:label>
+		<skos:definition xml:lang="en">strategy for calculating or otherwise determining an official closing price, typically by an exchange or trading venue</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;CollectionOfPrices">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-arr;DatedStructuredCollection"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-utl-alx;StatisticalPopulation"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-fi-ip;SecurityPrice"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">collection of prices</rdfs:label>
+		<skos:definition xml:lang="en">statistical population consisting of a series of prices for some security</skos:definition>
```

Comment:
The collection is used in various calculations, such as arithmetic means, but I've removed that since you don't like it (although I believe it is true); added a restriction for the security